docs/README.rst

```diff
@@ -1,49 +0,0 @@
-###################
```

Comment:
Do you plan to have a readme for the docs directory? It would be good to describe how this directory should be used. I can see that it might be confusing to have a README that doesn't get included in the generated documentation though.